Sources/Commands/SwiftPackageTool.swift

```diff
@@ -1061,8 +1061,7 @@ extension SwiftPackageTool {
             // Run the command plugin.
             let buildEnvironment = try swiftTool.buildParameters().buildEnvironment
             let _ = try tsc_await { plugin.invoke(
-                action: .performCommand(arguments: arguments),
-                package: package,
+                action: .performCommand(package: package, arguments: arguments),
```

Comment:
The package is pulled into the action to pave the way for package-less actions, e.g. for SwiftPM to query the plugin in some way (at some point to do something similar to SwiftArgParser and glean the parameters from the declaration of the plugin, though we're far from that yet — it's an illustration of the kind of thing that requires this factoring).